pkg/engine/renderer/dependencies.go

```diff
@@ -161,13 +171,35 @@ func sanitizeAndSerialize(origObj *unstructured.Unstructured) (string, error) {
 	// OwnerReferences need to be skipped as they contain a changing UID
 	obj.SetOwnerReferences([]metav1.OwnerReference{})
 
+	// These fields should not be present when we get the obj from our task or from the LastAppliedConfigAnnotation, but
+	// we may encounter objects that are not deployed by KUDO and therefore do not have the LastAppliedConfigAnnotation.
+	// This is a best effort attempt to make the hash as stable as possible
+	v := int64(0)
+	obj.SetResourceVersion("")
```

Comment:
I would be interesting in how to created this list
It is unclear to me if this "overkill" , "insufficient" or "just right"

Reply:
It's certainly overkill :) I basically went through all setters on Unstructured...